app/dashboard/templates/submit_bounty.html

```diff
@@ -110,9 +110,11 @@ <h1>{% trans "Fund an Issue" %}</h1>
                     <div class="form__select2">
                       <select name='deonomination' id='token'></select>
                     </div>
-                    <small class="form__input-help form__input-help--dynamic" id="usd_amount"></small>
                   </div>
                 </div>
+                <div>
+                  <small class="form__input-help form__input-help--dynamic" id="usd_amount"></small></form>
```

Comment:
> </form>

this should not be in there!

Reply:
the `</form>`

Reply:
:see_no_evil: My bad

Let me fix it

Reply:
@arun1595 All good.  Don't worry about it.  @owocki committed the fix:  https://github.com/gitcoinco/web/commit/100ed5f5a953f63d0fe5d15e012cc7d581da1c15

Reply:
@mbeacom Yeah, I noticed. Apology for being sloppy, won't happen again I assure.